package/src/components/Indicators/LoadingIndicator.tsx

```diff
@@ -21,14 +21,14 @@ const styles = StyleSheet.create({
 const LoadingIndicatorWrapper: React.FC<{ text: string }> = ({ text }) => {
   const {
     theme: {
-      colors: { black },
+      colors: { black, white_snow },
       loadingIndicator: { container, loadingText },
     },
   } = useTheme();
 
   return (
-    <View style={[styles.container, container]}>
-      <Spinner />
+    <View style={[styles.container, { backgroundColor: white_snow }, container]}>
+      <Spinner height={20} width={20} />
```

Comment:
`<Spinner height={20} width={20} />` is better as `<Spinner />`

Looking at this, it seems like the spinner component itself is bad to me.. 

sthe style by default has height and widdth of 16

but the icon doesnt have.. so its easier if we pass a default of 16 to the icon when no prop is passed and as a long term view, better